### PR TITLE
Enable ssl

### PR DESCRIPTION
**Pull request by Seven-Y-Q-Guo**
https://github.com/parvardegr/sharing/issues/19

As we're using express instead as server, just enable the ssl accordingly.

Screen Shot 2022-10-10 at 10 36 55 PM

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -192,15 +222,15 @@ var debugLog = (log) => {
         }
 
         if (options.receivePort)
-            app.listen(options.receivePort, listener);
```
Comment (parvardegr) [inline bin/index.js:195]: well done🙏
can you write a function for this? it duplicates four times in the code 
after that you can squash and merge it, also it’s good to add a TODO about creating a self-signed certificate if no cert and key provided.

Comment (parvardegr) [reply bin/index.js:195]: if you add
- support ssl
in the README it would be so nice

Comment (Seven-Y-Q-Guo) [reply bin/index.js:195]: > well done🙏
> can you write a function for this? it duplicates four times in the code 
> after that you can squash and merge it, also it’s good to add a TODO about creating a self-signed certificate if no cert and key provided.

sure, will work on it later today.

**Review by parvardegr — APPROVED** [review]
excellent

**Comment by parvardegr** [comment]
please use "squash and merge" when you want to merge this PR. (or other PRs :) )

**Comment by Seven-Y-Q-Guo** [comment]
> please use "squash and merge" when you want to merge this PR. (or other PRs :) )

Well noted.

**Pull request merged**